ansible/memcached-operator/watches.yaml

```diff
@@ -2,4 +2,4 @@
 - version: v1alpha1
   group: cache.example.com
   kind: Memcached
-  role: /opt/ansible/roles/Memcached
\ No newline at end of file
+  role: /opt/ansible/roles/memcached
```

Comment:
It doesn't look like you renamed the roles/Memcached directory to match this change

Reply:
I re-gen the project with the tool to ensure that it is == the current implementation and the dir is not created with upper case.